seed.js

```diff
@@ -1,57 +1,235 @@
-/*
+'use strict';
 
-This seed file is only a placeholder. It should be expanded and altered
-to fit the development of your application.
+/**
+ * run with node seed.js
+ */
 
-It uses the same file the server uses to establish
-the database connection:
---- server/db/index.js
+// dependencies
+var chalk = require( 'chalk' );
+var db = require( './server/db' );
+var Promise = require( 'sequelize' )
+  .Promise;
+const faker = require( 'faker' );
+const utils = require( './server/app/routes/utils' );
+const {
+  Credentials
+} = utils;
+const Sequelize = require( 'sequelize' );
 
-The name of the database used is set in your environment files:
---- server/env/*
+// binding to the models
+const models = db.models;
+const {
+  user: User,
+  organization: Organization,
+  userOrganization: UserOrganization,
+  userTeam: UserTeam,
+  team: Team,
+  annotation: Annotation,
+  location: Location,
+  template: Template,
+  assessment: Assessment,
+  criterionResponse: CriterionResponse,
+  question: Question,
+  questionResponse: QuestionResponse,
+  rubric: Rubric,
+  studentTest: StudentTest
+} = models
 
-This seed file has a safety check to see if you already have users
-in the database. If you are developing multiple applications with the
-fsg scaffolding, keep in mind that fsg always uses the same database
-name in the environment files.
 
-*/
+// console.log(models);
+// const User = db.models[ 'user' ];
+// const Organization = db.models[ 'organization' ];
+// const UserOrganization = db.models[ 'userOrganization' ];
 
-var chalk = require('chalk');
-var db = require('./server/db');
-var User = db.model('user');
-var Promise = require('sequelize').Promise;
+function createRandomCredentials( username, password, email ) {
+  let credentials = new Credentials(
+    username || faker.random.word(),
+    password || 'password',
+    faker.internet.exampleEmail()
+  )
+  credentials.username = credentials.username.replace( /\W/g, '' )
+    .toLowerCase()
+  credentials.photo = faker.internet.avatar();
+  credentials.name = faker.name.findName();
+  return credentials;
+}
 
-var seedUsers = function () {
+function randomN( n ) {
+  return Math.ceil( Math.random() * n )
+}
 
-    var users = [
-        {
-            email: 'testing@fsa.com',
-            password: 'password'
-        },
-        {
-            email: 'obama@gmail.com',
-            password: 'potus'
-        }
-    ];
+//seed methods
+/** seeds a random number of users 1-100 */
+const seedUsers = function ( n = Math.ceil( Math.random() * 100 ) ) {
```

Comment:
this input seems a lot like the function on line 57? Could you reuse the function?